src/helpers/files.js

```diff
@@ -241,15 +242,40 @@ function getAllFiles (projectRoot, dirPath, args, arrayOfFiles = []) {
         arrayOfFiles
       )
     } else {
-      arrayOfFiles.push(
-          `${path.join(dirPath.replace(projectRoot, '.'), file)}\n`
-      )
+      log(`Joining [${dirPath}] and [${file}] `, { level: 'debug', args })
+      const pathString = osJoin(projectRoot, dirPath, file, args.osPlatform)
+      log(`Adding [${pathString}] to the file list`, { level: 'debug', args })
+      arrayOfFiles.push(pathString)
     }
   })
   log(`Search complete for files in ${dirPath}`, { level: 'debug', args })
   return arrayOfFiles
 }
 
+/**
+ * Perform an os relevant join
+ * @param {string} projectRoot 
+ * @param {string} dirPath
+ * @param {string} file 
+ * @param {string} platform
+ * @returns {string}
+ */
+function osJoin(projectRoot, dirPath, file, platform) {
+  switch (platform) {
+    case 'linux':
+    case 'darwin':
+       return path.join(dirPath.replace(projectRoot, '.'), file)
+      break;
+    case 'win32':
+      const pathObj = path.parse(projectRoot)
+      console.log(pathObj)
+      return path.join(dirPath.replace(pathObj.base, ''), file)
```

Comment:
@eddiemoore What needs to happen here to change `C:\Users\circleci\project\src\ci_providers\index.js` into `src\ci_providers\index.js`

Project root: C:/Users/circleci/project
DirPath: src\ci_providers

I feel like I've tried everything.

Reply:
```js
path.join(dirPath.replace(projectRoot, ''), file)
```
?